doc/Learn/Maui/ThirdParty-EsriMaps.md

```diff
@@ -9,54 +9,96 @@ The MapView control, that's part of the ArcGIS Maps SDK for .NET, can be used in
-                    fonts.AddFont("MauiEmbeddingApp/Assets/Fonts/OpenSansSemibold.ttf", "OpenSansSemibold");
-                });
-    }
-    ```
+At this point you'll enter the **Uno Platform Template Wizard**, giving you options to customize the generated application.
+
+- Select **Blank** in **Presets** selection
+
+- Select the **Platforms** tab and unselect **Desktop** and **Web Assembly** platforms
+
+- Select the **Features** tab and click on **.NET MAUI Embedding**
 
-    > [!NOTE]
-    > If you have a license key and/or a location service API key, uncomment the `delegate` provided on `UseArcGISRuntime` method. This isn't required to run this sample.
+For more information on all the template options, see [Using the Uno Platform Template](xref:Uno.GettingStarted.UsingWizard).
+
+- Click **Create** to complete the wizard
+
+The template will create a solution with a single cross-platform project, named `ArcGisApp`, ready to run.
+
+### [Command Line](#tab/cli)
+
+> [!NOTE]
+> If you don't have the **Uno Platform dotnet new templates** installed, follow [dotnet new templates for Uno Platform](xref:Uno.GetStarted.dotnet-new).
+
+Create a new application using the `unoapp` template, enabling .NET MAUI Embedding. In this case, we're going to use the Blank template (`-preset blank`) and include .NET MAUI Embedding support (`-maui`).
+
+```bash
+dotnet new unoapp -preset blank -maui -platforms "android" -platforms "ios" -platforms "maccatalyst" -platforms "windows" -o ArcGisApp
+```
+
+This will create a new folder called **ArcGisApp** containing the new application.
+
+---
+
+## Nuget Packages
+
+Add a reference to the [Esri.ArcGISRuntime.Maui NuGet package](https://www.nuget.org/packages/Esri.ArcGISRuntime.Maui) to the `ArcGisApp.MauiControls` project.
+
+## Updating AppBuilderExtensions
+
+In the `AppBuilderExtensions` class, on `ArcGisApp.MauiControls` project, update the `UseMauiControls` extension method to call the `UseArcGISRuntime` method.
+
+```cs
+using Esri.ArcGISRuntime;
+using Esri.ArcGISRuntime.Maui;
+using Esri.ArcGISRuntime.Security;
+
+namespace ArcGisApp;
+
+public static class AppBuilderExtensions
+{
+    public static MauiAppBuilder UseMauiControls(this MauiAppBuilder builder) 
+        => builder
+            .UseArcGISRuntime(
+            //config => config
+            //    .UseLicense("[Your ArcGIS Maps SDK License key]")
+            //    .UseApiKey("[Your ArcGIS location services API Key]")
+            //    .ConfigureAuthentication(auth => auth
+            //        .UseDefaultChallengeHandler() // Use the default authentication dialog
+            //    )
+            )
+            .ConfigureFonts(fonts =>
+            {
+                fonts.AddFont("ArcGisApp/Assets/Fonts/OpenSansRegular.ttf", "OpenSansRegular");
+                fonts.AddFont("ArcGisApp/Assets/Fonts/OpenSansSemibold.ttf", "OpenSansSemibold");
+            });
+}
```

Comment:
I don't think you need the "ArcGisApp" part of the path of the font files. I created a new unoapp from the cli using the above arguments and it generated:

```cs
public static MauiAppBuilder UseMauiControls(this MauiAppBuilder builder) =>
    builder
        .ConfigureFonts(fonts =>
        {
            fonts.AddFont("Assets/Fonts/OpenSansRegular.ttf", "OpenSansRegular");
            fonts.AddFont("Assets/Fonts/OpenSansSemibold.ttf", "OpenSansSemibold");
        });
}
```

Reply:
True. This changed with the new single project structure.

I'll update the docs accordingly :)